test/nn/models/test_revgnn.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import torch
+
+from torch_geometric.nn import GroupAddRev, SAGEConv
+from torch_geometric.nn.dense.linear import Linear
+
+
+@pytest.mark.parametrize("num_groups", [2, 4, 8, 16])
+def test_revgnn(num_groups):
+    hidden_channels = 32
+    x = torch.randn(4, 32)
+    edge_index = torch.tensor([[0, 1, 2, 3], [0, 0, 1, 1]])
+    seed_gnn = SAGEConv(
+        hidden_channels // num_groups,
+        hidden_channels // num_groups,
+        aggr="mean",
+    )
+    gnn = GroupAddRev(seed_gnn, num_groups=num_groups, disable=False)
+    assert gnn.__repr__() == f"GroupAddRev(num_groups={num_groups}, \
```

Comment:
```suggestion
    assert str(gnn) == f"GroupAddRev(num_groups={num_groups}, \
```
Shouldn't it also print out `seed_gnn` as part of `__repr__`?